plugin/interestingwords.vim

```diff
@@ -197,29 +197,27 @@ function! s:buildColors()
   if (s:hasBuiltColors)
     return
   endif
-  if has('gui_running')
-    let ui = 'gui'
-    let wordColors = g:interestingWordsGUIColors
-  else
-    let ui = 'cterm'
-    let wordColors = g:interestingWordsTermColors
-  endif
+  let nGUIColors = len(g:interestingWordsGUIColors)
+  let nTermColors = len(g:interestingWordsTermColors)
+  let colorIndices = range(max([nGUIColors, nTermColors]))
```

Comment:
I think `g:interestingWordsGUIColors` and `g:interestingWordsTermColors` should not effect each other.

```
let g:interestingWordsGUIColors = ['#8CCBEA', '#A4E57E', '#FFDB72']
let g:interestingWordsTermColors = ['154', '121']
```

When I set the above config, I can highlights three words in terminal, and the first and third word have same color. It is different from the previous version.

![image](https://cloud.githubusercontent.com/assets/1998490/22235039/c17abbec-e237-11e6-9a0b-6fca018fb9c5.png)